docs/deployment/managed-scanning/github.md

```diff
@@ -1,78 +1,47 @@
 ---
-slug: managed-scanning
-title: Managed Scans (beta)
+slug: github
+append_help_link: true
+title: GitHub
 hide_title: true
-description: Semgrep Managed Scans provides an alternative to CI-based workflows. It enables you to add repositories to your Semgrep org in bulk without changing your CI workflows.
+description: Learn how to add a GitHub repository to Semgrep Managed Scans.
 tags:
   - Beta
   - Deployment
   - Semgrep AppSec Platform
 ---
 
-import InstallPrivateGitHubApp from "/src/components/procedure/_install-private-github-app.mdx"
-import DeleteAProject from "/src/components/procedure/_delete-a-project.md"
+import ScanWithSms from "/src/components/procedure/_scan-with-sms.mdx"
 
-# Semgrep Managed Scans
+# Add a GitHub repository to Semgrep Managed Scans
 
-Add repositories to your Semgrep organization in bulk without adding or changing your existing CI workflows through **Managed Scans**. Similar to CI workflows, Managed Scans also integrates into developer workflows through PR comments.
```

Comment:
Any reason why we're removing the bit about PR comments? I think removing line 19 ("Alternative method to CI...") is ok, but how do we convey (as concisely as possible) that SMS is not a downgrade from CI in terms of feature availability?